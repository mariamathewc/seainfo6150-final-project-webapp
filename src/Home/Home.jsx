import React from 'react'
import Form from "../Form/Form";
import treefrog from "../images/treefrog.jpg";

const Home = () => {
    return (
        <div>
            The home page
            <h1>I changed this homepage</h1>
            <img src={treefrog} alt="tree frog" />
            <Form />
        </div>
    )
}

export default Home
