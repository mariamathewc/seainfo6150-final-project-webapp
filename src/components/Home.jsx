import React from 'react';
import styles from "./Home.module.css";
import { NavLink } from 'react-router-dom';


	const Home = (props) => {
	

        return (




			<div className={styles.container}>
				<div className={styles.home}>
					<h2>Travel in Tents</h2>
					<br />
					<p>Welcome to Travel in Tents</p>
					<p>Jump right in and explore our campgrounds</p>
					<p> Checkout our exclusive outdoor equipments to make your camping fun</p>

					<NavLink className={styles.footer} to='/about' exact activeClassName='active'  > <b>About </b> </NavLink>
					<NavLink className={styles.footer} to='/contacts' exact activeClassName='active'  > <b>Contacts </b> </NavLink>


				</div>
			</div>

            

		)
	
}

export default Home 