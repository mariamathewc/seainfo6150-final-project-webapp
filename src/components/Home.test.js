import React from "react";
import Home from "./Home.jsx";
import { render } from "@testing-library/react";
import { BrowserRouter } from 'react-router-dom';



describe("Home tests", () => {
	it("renders correctly", () => {
		const { container } = render(<BrowserRouter><Home /></BrowserRouter>);
		expect(container).toMatchSnapshot();
	});
});