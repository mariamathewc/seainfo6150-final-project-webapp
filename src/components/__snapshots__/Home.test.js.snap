// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home test renders correctly 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="home"
    >
      <h2>
        Travel in Tents
      </h2>
      <br />
      <p>
        Welcome to Travel in Tents
      </p>
      <p>
        Jump right in and explore our campgrounds
      </p>
      <p>
         Checkout our exclusive outdoor equipments to make your camping fun
      </p>
      <a
        class="footer"
        href="/about"
      >
         
        <b>
          About 
        </b>
         
      </a>
      <a
        class="footer"
        href="/contacts"
      >
         
        <b>
          Contacts 
        </b>
         
      </a>
    </div>
  </div>
</div>
`;
